model/docs/pyrenew_demo.qmd

```diff
@@ -49,8 +49,13 @@ with seed(rng_seed=np.random.randint(0,1000)):
     q_samp = q.sample(duration=100)
 
 plt.plot(np.exp(q_samp[0]))
```

Comment:
Good question, we need to document what is the output of this function. Every sample function from `pyrenew` returns a tuple or named tuple.